playground/dist/index.html

```diff
@@ -25,6 +27,11 @@
         background-color: rgb(38, 101, 163);
       }
 
+      .query-builder *,
+      .data-viewer * {
+        box-sizing: border-box;
+      }
```

Comment:
Can we do even better and add
```
/deep/ * {
  box-sizing: border-box;
}
```
in both DataViewer and QueryBuilder components?

Reply:
c'est ce que j'avais fait à la base mais dans tucana on va avoir un doublon au niveau de la déclaration du box-sizing (tout les éléments ont déja `box-sizing: border-box` ). 

On avait dis qu'on ne souhaitait pas faire du `* {  something }` dans le code de sortie de weaverbird.

Peut être qu'on ne s'est pas compris ?